charts/rancher-gatekeeper-crd/102.0.0+up3.10.0/templates/jobs.yaml

```diff
@@ -40,6 +40,13 @@ spec:
             - name: crd-manifest
               readOnly: true
               mountPath: /etc/config
+          securityContext:
+            seccompProfile:
```

Comment:
shouldn't all of these be gated by `.Values.enableRuntimeDefaultSeccompProfile`?

Reply:
I think a broader issue here might also be that our UI does not provide the capability to update values for a CRD chart on install. We might need to open a UI ticket for this.

But in general, if you were to deploy this on a cluster whose nodes do not use `seccomp`, wouldn't you run into issues? Has this been tested?

Reply:
updated the values for crd chart as per your suggestion but as you mentioned UI related work will be needed so that user can modify these values for CRD chart.

Reply:
For context, k8s 1.25+ has the seccomp `RuntimeDefault` profile set as default to all pods. So this may not be needed in the feature.

> But in general, if you were to deploy this on a cluster whose nodes do not use seccomp, wouldn't you run into issues?

AFAIK, seccomp would fail open - if it is not supported (e.g. Windows nodes) it will just be ignored.